mobile-app/app/screens/AppNavigator/screens/Loans/components/AddOrRemoveCollateralForm.tsx

```diff
@@ -157,17 +159,48 @@ export const AddOrRemoveCollateralForm = React.memo(({ route }: Props): JSX.Elem
           />
         </ThemedView>
       </WalletTextInput>
+
       <InputHelperText
         label={`${translate('components/AddOrRemoveCollateralForm', isAdd ? 'Available' : 'Current')}: `}
         content={available}
         testID='form_balance_text'
-        suffix={` ${token.displaySymbol}`}
+        suffixType='component'
         styleProps={tailwind('font-medium')}
-      />
-      {isConversionRequired &&
+      >
+        <ThemedText
+          light={tailwind('text-gray-700')}
+          dark={tailwind('text-gray-200')}
+          style={tailwind('text-sm font-medium')}
+        >
+          <Text>{' '}</Text>
+          {token.displaySymbol}
+          {
+            !new BigNumber(activePrice).isZero() && (
+              <NumberFormat
+                value={activePrice.multipliedBy(available).toFixed(2)}
+                thousandSeparator
+                decimalScale={2}
+                displayType='text'
+                prefix='$'
+                renderText={(val: string) => (
+                  <ThemedText
+                    dark={tailwind('text-gray-400')}
+                    light={tailwind('text-gray-500')}
+                    style={tailwind('text-xs')}
```

Comment:
Fix for alignment in android:
```suggestion
                    style={tailwind('text-xs leading-5')}
```